src/vs/workbench/contrib/tasks/browser/abstractTaskService.ts

```diff
@@ -82,6 +82,12 @@ import { isWorkspaceFolder, TaskQuickPickEntry, QUICKOPEN_DETAIL_CONFIG, TaskQui
 import { ILogService } from 'vs/platform/log/common/log';
 import { once } from 'vs/base/common/functional';
 
+// {{ SQL CARBON EDIT }}
+// integration with tasks view panel
+import { ITaskService as ISqlTaskService, TaskStatusChangeArgs } from 'sql/workbench/services/tasks/common/tasksService';
+import { TaskStatus } from 'sql/workbench/api/common/extHostBackgroundTaskManagement';
+import { TaskInfo } from 'azdata';
```

Comment:
is it okay to ref a azdata item from VS tree - just checkin if it would cause any code layering warnings to show up?

Reply:
this happens to be an interface only, not recommended but works since at runtime it will be ignored. azdata module doesn't exist here at runtime.